tests/integration_tests/test_matchmaker.py

```diff
@@ -62,7 +60,7 @@
 
 async def test_matchmaker_info_message(lobby_server, mocker):
     mocker.patch('server.matchmaker.matchmaker_queue.time', return_value=1_562_000_000)
-    mocker.patch('server.matchmaker.matchmaker_queue.config.QUEUE_POP_TIME_MAX', return_value=1)
+    mocker.patch('server.matchmaker.matchmaker_queue.config.QUEUE_POP_TIME_MAX', 1)
```

Comment:
🤔 
Why remove the additional precision for the unskilled reader?

Reply:
This actually changes the functionality. If you specify `return_value` then it will patch the object with an instance of `Mock` that when called as a function, returns that value. However if you supply it as a positional arg, then it actually replaces the object you are patching, with that new object. So...

Before (with `return_value=1`):
```python
QUEUE_POP_TIME_MAX = Mock(return_value=1)
```
After (with `1` as positional arg)
```python
QUEUE_POP_TIME_MAX = 1
```

The problem was that another part of the code was trying to compare `QUEUE_POP_TIME_MAX` and it was throwing some error like 'Mock object doesn't support < comparison with float'